src/prefect/cli/orion.py

```diff
@@ -150,6 +152,41 @@ async def start(
     app.console.print("Orion stopped!")
 
 
+@orion_app.command()
+async def status(
+    wait: bool = typer.Option(
+        False,
+        "--wait",
+        help="Runs health check loop until successful or optional timeout value is exceeded.",
+    ),
+    timeout: int = typer.Option(
+        None, "--timeout", help="Optional Timeout value in seconds"
+    ),
+):
+    """Verify Connection status with Orion API"""
+    sleep_time = 1  # Initial sleep time when polling
+    async with get_client() as client:
+        with (
```

Comment:
Let's do some sort of display about what server we're connecting to?

Reply:
Would is make sense to display the api URL we're attempting to connect to here?

Reply:
That could work! You'll need to consider the ephemeral case as well but that should always be available (it might fail a database migration or something on startup so a status check is reasonable).

Reply:
I've made some changes here to display the URL we're attempting to connect to, as for the ephemeral api is there a good way to check for that and would we just want to exit the status check at that point?

Reply:
I ended up including a check to exit the cli command if it's running against the ephemeral API with a message indicating that the PREFECT_API_URL needs to be set in order to verify the status of an Orion or Cloud api connection